packages/lit-starter-js/package.json

```diff
@@ -34,8 +34,7 @@
   "author": "Google LLC",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "lit": "^2.0.0-rc.1",
-    "lit-element": "^3.0.0-rc.1"
```

Comment:
I think `lit-element` had to stay a dependency due to one of the 3rd party tools erroring without it. @kevinpschaaf ?

Reply:
Right, `@open-wc/testing` has an undeclared peer dependency which breaks the TypeScript build depending on how npm optimizes the package install; see https://github.com/open-wc/open-wc/issues/2085. Can be removed once that issue is resolved.